Update release notes (#34)
tidbcloud/dbaas-docs: release-notes.md

This page lists the release notes of [TiDB Cloud](https://pingcap.com/products/tidbcloud).

## December 30, 2020

* Upgrade the default TiDB version to v4.0.9
* Support upgrading and scaling in TiDB gracefully to achieve zero client failures

> * Recover cluster configuration after restoring new cluster from backup

Reviewer: ```suggestion
* Recover cluster configuration after restoring a new cluster from backup
```

## December 16, 2020

> * Adjust the minimum TiDB nodes to one for all cluster tiers

Reviewer: ```suggestion
* Adjust the minimum number of TiDB nodes to one for all cluster tiers
```

* Prohibit executing system command on the SQL web shell

> * Enable redact-log for TiDB cluster by default

Reviewer: ```suggestion
* Enable redact-log for a TiDB cluster by default
```

## November 24, 2020

> * Allow TiDB cluster public endpoint’s traffic filter IP list to be empty to disable public access

Reviewer: To make it easier to read.
```suggestion
* Allow the traffic filter IP list of a TiDB cluster's public endpoint to be empty to disable public access
```

> * Improve invitation email delivery rate to customers with outlook or hotmail

Reviewer: ```suggestion
* Improve the delivery rate of invitation emails sent to customers with Outlook or Hotmail
```

> * Polish error notification experience for sign up

Reviewer: ```suggestion
* Polish the error notification message for sign-up
```

* New clusters will run on CentOS VM instead of Ubuntu

> * Fix cluster not showing in recycle bin when the corresponding backup is still exist

Reviewer: ```suggestion
* Fix the issue that the cluster does not show in the recycle bin when the corresponding backup still exists
```
